.travis.yml

```diff
@@ -112,7 +113,11 @@ install:
   - python -m pip install -U pip
   - python -m pip --version
   - python -m easy_install -U setuptools
-  - python -m pip install numpy scipy networkx pytest python-igraph jax jaxlib torch
+  - python -m pip install numpy scipy networkx pytest python-igraph jax jaxlib
+  - |
+    if [ -n "$CONDA" && $TRAVIS_OS_NAME == "osx" ]; then
```

Comment:
this disables tests also on linux though? you should test on conda on mac/os + all linux distributions, right?